tools/webpack/packages.js

```diff
@@ -139,6 +140,22 @@ module.exports = {
 			}
 			return `webpack://${ info.namespace }/${ info.resourcePath }`;
 		},
+		// This is the default, but required for @shopify/web-worker.
+		globalObject: 'self',
+	},
+	module: {
+		rules: [
+			...baseConfig.module.rules,
+			{
+				test: /\.wasm$/,
+				type: 'asset/resource',
+				generator: {
+					// FIXME: Do not hardcode path.
+					filename: './build/vips/[name].wasm',
+					publicPath: '',
```

Comment:
This may be alright, maybe it should go under `build/vendor/[name].wasm` and it should match wasm in the `wasm-vips` package.

Reply:
Moving to `build/vendors` feels wrong though 🤔 

Especially since it should also be in `packages/vips/build` and `packages/vips/build-module`